src/client/components/LogView.tsx

```diff
@@ -59,6 +66,48 @@ class LogView extends Component<LogViewProps, LogViewState> {
     })
   }
 
+  renderCollapseButton = (): JSX.Element => {
+    if (this.state.collapsed === true) {
+      return (
+        <CollapseButton
+          label="Expand"
+          onClick={() => this.handleCollapseChange(false)}
+        />
+      )
+    } else {
+      return (
+        <CollapseButton
+          label="Collapse"
+          onClick={() => this.handleCollapseChange(true)}
+        />
+      )
+    }
+  }
+
+  renderJsonObj = (): JSX.Element => {
```

Comment:
Instead of this function you could just add this component directly to render:

```
<ReactJson
        src={this.state.log}
        name="Log"
        theme="monokai"
        displayDataTypes={false}
        collapsed={this.state.collapsed ? 2 : 5}
        style={{
          margin: '0 auto',
          width: '100%',
          minHeight: '30vh',
          maxHeight: '90vh',
          overflow: 'auto',
          whiteSpace: 'pre-line'
        }}
      />
```